docs/buildah-manifest-rm.md

```diff
@@ -0,0 +1,25 @@
+# buildah-manifest-rm "1" "April 2021" "buildah"
+
+## NAME
+buildah\-manifest-rm - Removes one or more manifest lists.
+
+## SYNOPSIS
+**buildah manifest-rm** [*list* ...]
+
+## DESCRIPTION
+Removes one or more locally stored manifest lists.
+
+## EXAMPLE
+
+buildah manifest rm listID
```

Comment:
Is it supporting only IDs? I think we can also pass a list name, right? So how about:
```suggestion
buildah manifest rm <list>
```
(or any other placeholder)

Reply:
fixed